spec/models/external_service/survey_response_spec.rb

```diff
@@ -83,6 +83,24 @@
       receiver = survey_response.save!
       expect(receiver.receiver.quantity).to eq quantity
     end
+
+    it "locks the quantity for order if there is a quantity param" do
+      params[:quantity] = order_detail.quantity
+      receiver = survey_response.save!
+      expect(order_detail.reload).to be_quantity_locked_by_survey
```

Comment:
Not really any reason to assign `receiver`. I think that's just a holdover from copying and pasting some setup. I'll change that as well as to using the `expect {}.to change`. I like that better as well.